pkgs/applications/video/gnome-twitch/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, meson, cmake, ninja, wrapGAppsHook
+, libpeas, gtk3, libsoup, json-glib, webkitgtk, glib, glib-networking
+, cairo, clutter-gtk, clutter-gst, epoxy, gstreamer, mpv
+, python3, desktop-file-utils
+, substituteAll
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-twitch-unstable";
+  version = "2019-05-20";
+
+  src = fetchFromGitHub {
+    owner = "vinszent";
+    repo = "gnome-twitch";
+    rev = "8e774a04fdd15fa4c9990918bf89295a232d1715";
+    sha256 = "0pvxfwbv4fms2d0sp43jw83kp1i6gq03kg6gwy8b1fhp473gxrfj";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson cmake ninja wrapGAppsHook python3 ];
```

Comment:
where's cmake needed?

Reply:
Meson tries to find CMake in case some dependency needs to be found using it but usually, `pkg-config` is enough.